src/main/groovy/de/dkfz/roddy/execution/jobs/cluster/lsf/rest/LSFRestJobManager.groovy

```diff
@@ -194,11 +214,14 @@ class LSFRestJobManager extends BatchEuphoriaJobManagerAdapter {
         RestResult result = restExecutionService.execute(new RestCommand(URI_JOB_SUBMIT, requestBody.toString(), headers, RestCommand.HttpMethod.HTTPPOST)) as RestResult
         if (result.statusCode == 200) {
             logger.postAlwaysInfo("status code: " + result.statusCode + " result:" + new XmlSlurper().parseText(result.body))
-            job.setRunResult(new BEJobResult(job.lastCommand, new BEJobID(new XmlSlurper().parseText(result.body).text(), job), result.successful, job.tool, job.parameters, job.parentJobs as List<BEJob>))
```

Comment:
What? Why? Haeh? I am not going to pull the XMLSlurper stuff into BEJob, if you mean that. This would integrate different application levels too much.

Reply:
Haha no, I did not say that :D but: new XmlSlurper().parseText(result.body). is done three times. Can be put into a variable with a good name.

Reply:
At this place it is twice. Now substituted by variable. The other places parse other output.